extra-js/bitwarden-cli/spec

```diff
@@ -0,0 +1,7 @@
+VER=1.15.1
+JSLIBCOMMIT="f80e89465ffc004705d2941301c0ffb6bfd71d1a"
+SRCS="tbl::https://github.com/bitwarden/cli/archive/v$VER.tar.gz \
+      tbl::https://github.com/bitwarden/jslib/archive/${JSLIBCOMMIT}.tar.gz"
```

Comment:
You should use a Git source here instead, saving an extra variable that could be easily ignored.